definitions/npm/promise-limit_v2.x.x/test_promise-limit_v2.x.x.js

```diff
@@ -0,0 +1,16 @@
+// @flow strict
+
+import limit from "promise-limit";
+
+// $ExpectError
+limit("");
+
+const limiter = limit(3);
+
+function dosomething_(): Promise<string> {
+  return Promise.resolve("hi");
+}
+const dosomething = () => limiter(() => dosomething_());
+(dosomething: () => Promise<string>);
+
+dosomething().then(console.log);
```

Comment:
Bonus point for [wrapping tests in `describe`, `it`](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#writing-tests) if you wish.

Reply:
Thanks for the suggestion - added in ac3c21ff0d7537f5861728133b865378d608bda4!